### ISSUE TITLE
Suggestion: Most used apps by user

### ISSUE THREAD

**Issue by iainmwallace** [RinteRface/connectViz#1]
Very nice package/concept. 

One dataset that might useful would be to able to view what content a specific user is consuming. 
For example, you might be interested in knowing how often a key stakeholder uses app 1 vs. app 2

**Comment by DivadNojnarg** [comment]
Indeed, this sounds like a good addition. I have to admit this first "release" was mostly focused on apps usage and developers rather than consumers and what they do. I am sure there are other stories to tell ...
